train_management/migrations/0019_auto_20210712_1724.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 3.1.6 on 2021-07-12 15:24
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('train_management', '0018_merge_20210711_1708'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Do you really want to remove `paid` and add `billed` or did you try to rename the existing field? I don't know by heart, but there should be an option to migrate with rebaming as well to keep the values

Reply:
For this change it doesn't matter, since there is no relevant info jn this field so far. For future changes it might be worth looking at these possibilities.